src/middleware.ts

```diff
@@ -1,5 +1,7 @@
 import { PLATFORM } from "aurelia-pal";
 
+const DEFAULT_LOCAL_STORAGE_KEY = "aurelia-store-state";

```

Comment:
since we're at it I think we should also export this const

Reply:
Good point, exported!